source/adminguide/networking/dns_and_dhcp.rst

```diff
@@ -17,6 +17,13 @@
 DNS and DHCP
 ------------
 
-The Virtual Router provides DNS and DHCP services to the guests. It
-proxies DNS requests to the DNS server configured on the Availability
-Zone.
+The Virtual Router & ConfigDrive (since Apache CloudStack 4.20) provides
+DNS and DHCP services to the guests. It proxies DNS requests to the DNS
+server configured on the Availability Zone.
+
+.. note::
+   In case of a network with ConfigDrive, adding/removing nic to/from an
+   instance or updating the ip address of a nic will not be reflected in the
+   instance if the instance is already running. To do so, run
```

Comment:
@vishesh92 
if the instance is stopped, will the ip address be updated when it is started ?

Reply:
It won't as of now. Let me make it more specific here.

Reply:
This is a limitation of ConfigDrive with cloud-init. It applies network configuration from ConfigDrive only during the first boot.

Reply:
I was definitely able to see the changes when adding/removing NICs only by executing stop/start VM sequence (not restart).